### ISSUE TITLE
fine-tuned I2V model generates corrupted video

### ISSUE THREAD

**Issue by Yuancheng-Xu** [THUDM/CogVideo#374]
Hi I fine-tuned a I2V model and encounter a problem: the generated video of the fine-tuned model is corrupted (almost noise).

To replicate it:
1. I fine-tune the I2V model using **zero** learning rate, so that it does not actually update the parameters. I use the saved model to do inference, and get a video of almost noise (it has some pattern of the provided reference image in the first few frames, but mostly noise).
2. I use the same inference code and configuration file and use the original I2V model, and it works as expected. 

My guess is that there could something wrong with the saved model, but I am not sure. Could you take a look? Thanks!

**Comment by zRzRzRzRzRzRzR** [comment]
did you use sat right, we will check. I just upload the diffusers version of I2V

**Comment by Yuancheng-Xu** [comment]
Yes I used SAT

**Comment by awwwq** [comment]
same problem here, finetune i2v with lora generate noise video

**Comment by Yuancheng-Xu** [comment]
Wondering if there are updates on this

**Comment by zhenghao977** [comment]
same problems

**Comment by Yuancheng-Xu** [comment]
Following up on the updates

**Comment by zRzRzRzRzRzRzR** [comment]
Will Open tomorrow in CogVideoX-Factory

**Comment by zRzRzRzRzRzRzR** [comment]
In this new repository, you can directly use the T5 and VAE build number datasets first, and then directly train the diffusers version of the transformers model
